garage/np/algos/base.py

```diff
@@ -5,3 +5,13 @@ class Algorithm:
 class RLAlgorithm(Algorithm):
     def train(self):
         raise NotImplementedError
+
+    def process_samples(self, itr, paths):
+        """
+        Return processed sample data (typically a dictionary of concatenated
+        tensors) based on the collected paths.
+        :param itr: Iteration number.
```

Comment:
I think we are going to use Google's Python docstring style. Please refer to this [template](https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html).